migrations/1564493063114-persist_files_by_copied_courses.js

```diff
@@ -265,15 +261,12 @@ const cloneMissingFilesAndUpdateLessons = (missingFileInfos, out) => {
 	const tasks = [];
 	missingFileInfos.forEach(({ target, sourceFile: file, teacher }) => {
 		// with the condition that we have at the moment only awsS3 as strategie, it is not check for every school.
-		const {
-			content, lessonId, index, courseId: parent,
-		} = target;
+		const { content, lessonId, index, courseId: parent } = target;
 		const id = file._id;
 		if (!teacher) {
 			out.pushFail(
 				lessonId,
-				'Skip fix file. No teachers in course exist.'
-				+ `Owner can not set. {fileId:${id}, lessonId:${lessonId}}`,
+				'Skip fix file. No teachers in course exist.' + `Owner can not set. {fileId:${id}, lessonId:${lessonId}}`
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)